fuzzing/private/oss_fuzz/package.bzl

```diff
@@ -15,26 +15,47 @@
 """Rule for packaging fuzz tests in the expected OSS-Fuzz format."""
 
 load("//fuzzing/private:binary.bzl", "FuzzingBinaryInfo")
+load("//fuzzing/private:util.bzl", "runfile_path")
 
 def _oss_fuzz_package_impl(ctx):
     output_archive = ctx.actions.declare_file(ctx.label.name + ".tar")
     binary_info = ctx.attr.binary[FuzzingBinaryInfo]
 
-    action_inputs = [binary_info.binary_file]
+    binary_runfiles = binary_info.binary_runfiles.files.to_list()
+    archive_inputs = binary_runfiles
+
+    runfiles_manifest = ctx.actions.declare_file(ctx.label.name + "_runfiles")
+    runfiles_manifest_content = "".join([
+        "{runfile_path} {real_path}\n".format(
+            real_path = runfile.path,
+            runfile_path = runfile_path(ctx, runfile),
+        )
+        for runfile in binary_runfiles
+        if runfile != binary_info.binary_file
```

Comment:
This is an important detail that might be easy to miss: Consider adding a comment here explaining why we skip the binary file in the iteration.